<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>639</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>639</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/elements/login.png);</string>
  </property>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/regist.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="RegistButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/loginButton.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UsernameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 24pt &quot;Baoli SC&quot;;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;用户名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameLine">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Baoli SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordLine">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/elements/frame.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoginLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Baoli SC&quot;;
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:64pt; font-weight:700; color:#000000;&quot;&gt;注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/back.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
